### DISCUSSION TITLE
console_title_template - How do I map home directory to ~

### DISCUSSION THREAD

**Discussion by peter-dolkens** [JanDeDobbeleer/oh-my-posh#340] [Troubleshoot] [answered]
Title says it all - I'm looking to clone my oh-my-zsh config, which currently looks like this:

```
[host.domain.com] ~/Folder/Path
```

For now I've hardcoded the hostname, but now I'm trying to see if it's possible to somehow do a `mapped_locations` or similar equivalent for the `.Path` variable.

Alternatively, could we get something like `sprig` template functions added so we can manipulate the title a bit?

**Comment by tradiff** [comment, marked as answer]
I created a PR for this: #343

**Comment by lnu** [comment]
I guess we can mark it as answered?

**Reply by peter-dolkens** [reply]
Yup - I had a hiccup on original testing, but a complete restart of powershell seems to have cleared things up now.

Thanks all!
